lib/bloc/uiDateManager/ui_date_manager_bloc.dart

```diff
@@ -32,4 +46,34 @@ class UiDateManagerBloc extends Bloc<UiDateManagerEvent, UiDateManagerState> {
       LogOutUiDateManagerEvent event, Emitter emit) async {
     emit(LoggedOutUiDateManagerUpdated());
   }
+
+  void handleDateChange(UiDateManagerUpdated state,
+      CarouselController dayRibbonCarouselController, DateTime date) {
```

Comment:
This code block should not be here. You should just need to emid a DateChangeEvent as can be seen in this [line](https://github.com/jerbio/tilerMobileApp/blob/49ae2f0c7315225bc18a5b65045fde1d57fca5c9/lib/components/tileUI/emptyDayTile.dart#L76)